src/Helper/IrisHelper.php

```diff
@@ -3,6 +3,7 @@
 namespace ElevenMiles\Iris\Helper;
 
 define('IRIS_ASSET_PATH', get_site_url() . '/vendor/eleven-miles/iris/src/assets');
+define('IRIS_ACTION_SCHEDULER_PATH', get_site_url() . '/vendor/eleven-miles/iris/libraries/action-scheduler/action-scheduler.php');
```

Comment:
Same as above comment, very happy to discuss how this is used as appreciate there is additional context for the functionality required here.